sdk/storage/storage-blob/rollup.base.config.js

```diff
@@ -53,7 +53,11 @@ export function nodeConfig(test = false) {
       }),
       nodeResolve({ preferBuiltins: true }),
       cjs()
-    ]
+    ],
+    onwarn(warning, warn) {
+      if (warning.code === "CIRCULAR_DEPENDENCY") throw new Error(warning.message);
+      warn(warning);
```

Comment:
What's the effects for this?

Reply:
This captures `CIRCULAR_DEPENDENCY` warning and throws an error instead.
For the rest of the rollup warnings, `warn(warning);` will help throw them as warnings.